reactapp/src/components/MakerTile/MakerTile.css

```diff
@@ -0,0 +1,10 @@
+.project {
```

Comment:
I was under the impression we should be using styled-components to style our components.

Reply:
oh ok, i think we can make changes to the design once we finalize that?